ambry-frontend/src/main/java/com/github/ambry/frontend/NamedBlobPutHandler.java

```diff
@@ -70,11 +70,11 @@
  * The body of the request should be a JSON object that conforms to the format described in {@link StitchRequestSerDe}.
  */
 public class NamedBlobPutHandler {
-  private static final Logger LOGGER = LoggerFactory.getLogger(NamedBlobPutHandler.class);
   /**
    * Key to represent the time at which a blob will expire in ms. Used within the metadata map in signed IDs.
    */
   static final String EXPIRATION_TIME_MS_KEY = "et";
+  private static final Logger LOGGER = LoggerFactory.getLogger(NamedBlobPutHandler.class);
```

Comment:
why do we have this kind of change? did you formatter do this for you?

Reply:
fixed. yeah formatter messes up something.